project/Dependencies.scala

```diff
@@ -5,9 +5,9 @@ object Dependencies {
     "junit" % "junit" % "4.12" % Test,
     ("com.novocode" % "junit-interface" % "0.11" % Test).exclude("junit", "junit-dep")
   )
-
+  
   object ideProbe {
-    val version = "0.4.0"
+    val version = "0.3.0+56-19245d51-SNAPSHOT"
```

Comment:
Why?

Reply:
Versioning of snapshots has some issues. I hope since 0.0.5 release we will have correct versioning again. The version that Tomek put here is actually newer.